doc/modules/manifold.rst

```diff
@@ -513,12 +514,16 @@ divergence could increase during this phase. Usually it does not have to be
 tuned. A critical parameter is the learning rate. If it is too low gradient
 descent will get stuck in a bad local minimum. If it is too high the KL
 divergence will increase during optimization. More tips can be found in
-Laurens van der Maaten's FAQ (see references).
+Laurens van der Maaten's FAQ (see references). The last parameter, angle,
+is a tradeoff between performance and accuracy. Larger angles imply that we
+can approximate larger regions by a single point,leading to better speed
+but less accurate results. 
 
-Standard t-SNE that has been implemented here is usually much slower than
+The Barnes-Hut t-SNE that has been implemented here is usually much slower than
 other manifold learning algorithms. The optimization is quite difficult
-and the computation of the gradient is on :math:`O[d N^2]`, where :math:`d`
-is the number of output dimensions and :math:`N` is the number of samples.
+and the computation of the gradient is on :math:`O[d N log(N)]`, where :math:`d`
+is the number of output dimensions and :math:`N` is the number of samples. This
+improves on the standard t-SNE which is :math:`O[d N^2]`. 
```

Comment:
This paragraph does not make it explicit that both methods are implemented in scikit-learn. I think it's important to state the pro and the cons of each method. In particular:

- BH is significantly more scalable In practice BH can be used to embed hundred of thousand of data points while the standard can barely handle thousands of samples,
- BH only works when the target dimensionality is 3 or less (typically 2 for a visualization with a matplotlib scatter plot),
- BH is an approximation of the standard method. The approximation is parameterized with the `angle` parameter, therefore the `angle` parameter is unused when `method="standard"`
- BH only works with dense input data. Sparse data matrices can only be embedded with the standard method.

For visualization purpose (which is the main use case of t-SNE), using the BH method is strongly recommended. The standard t-SNE method is useful for checking the theoretically properties of the embedding possibly in higher dimensional space but limit to small datasets due to computational constraints.